.github/workflows/ci.yml

```diff
@@ -0,0 +1,60 @@
+name: CI/CD
```

Comment:
Maybe I'm misunderstanding this, but I thought the idea was to have the CI build in the cloud? Is that called "self-hosted" in this file? Or are we trying to do the CI build on Daniel's server?

Reply:
@Andrew-Dickinson  We use the self hosted runner to execute code on Daniel's server, mostly the Deploy job.  I can run the Test and Build jobs on GitHub hosted runners if you want.  For our case I don't think it really matters, but I could see that being safer if the service was really important.

Reply:
@Andrew-Dickinson I changed it to GitHub hosted runners for testing and building.  I [tested it](https://github.com/andybaumgar/nycmesh-support-bot/actions/runs/2985790994) and every seems to work.  The Test and Build steps are a but slower, but if we actually care about speeding them up there are probably lots of other optimizations.